rmgpy/data/kinetics/family.py

```diff
@@ -3868,3 +3872,82 @@ def getObjectiveFunction(kinetics1,kinetics2,obj=informationGain,T=1000.0):
     
     return obj(ks1,ks2), N1 == 0
 
+################################################################################
+
+class ArrheniusRMGObject(RMGObject):
+    """
+    Child class of RMG Object for storing filter Arrhenius fits.
+    """
+
+    def __init__(self, unimol=None, bimol=None):
+        if unimol:
+            self.unimol = unimol
+        else:
+            self.unimol = {}
+        if bimol:
+            self.bimol = bimol
+        else:
+            self.bimol = {}
+
+    def load_yaml(self, path, class_dictionary):
+        with open(path, 'r') as f:
+            data = yaml.safe_load(stream=f)
+
+        data = {key:data[key] for key in data if key!='class'}
+        args = recursive_make_object(data, class_dictionary)
+        self.__init__(**args)
+
+
+def loadFilterFits(self):
```

Comment:
My first impression is that this should be a method of the KineticsDatabase class. Let me know if you considered that and whether there's a reason why that wouldn't work.

Reply:
Good idea, loadFilterFits is now a method of the KineticsDatabase class. Where would you recommend to include the unit-test (currently in familyTest.py; do we want to start a databaseTest.py in this folder)?